Cargo.toml

```
@@ -13,14 +13,23 @@ keywords = ["integer", "u256", "ethereum"]
 categories = ["cryptography::cryptocurrencies", "mathematics", "no-std"]
 
 [workspace]
-members = [
-  "bench",
-  "fuzz",
-  "intrinsics",
-]
+members = ["bench", "fuzz", "intrinsics"]
 
 [features]
+alloc = []
```

If this is only for `serde`, then I imagine this should also:
```suggestion
alloc = ["crate-serde/alloc"]
```

That being said, I don't think the `visit_string` implementation is needed - in which case this can likely be removed.